packages/btc/tsconfig.json

```diff
@@ -1,20 +1,19 @@
 {
   "compilerOptions": {
-    "moduleResolution": "Node",
+    "moduleResolution": "Bundler",
```

Comment:
The TypeScript document recommends using `"moduleResolution": "Node16"` instead of `"moduleResolution": "Bundler"` for writing a library. 

See: https://www.typescriptlang.org/docs/handbook/modules/guides/choosing-compiler-options.html#im-writing-a-library